tests/core/rw/rw.cpp

```diff
@@ -334,10 +334,10 @@ TEST_F(rw, 2)
 	    {}}));
 
 	if (env_block_size == 4096)
-		EXPECT_TRUE(test_pmemfile_stats_match(pfp, 2, 0, 609, 0,
+		EXPECT_TRUE(test_pmemfile_stats_match(pfp, 2, 0, 0x32c, 0,
```

Comment:
When the test failed, it was printed in hex, something like
0x261 != 0x32c
I did not see a reason to convert it to decimal. Now it is in the same format, as a failed test's output would be.